ci-operator/config/openshift/origin/openshift-origin-master.yaml

```diff
@@ -261,3 +261,7 @@ tests:
   commands: test/extended/conformance-k8s.sh
   openshift_installer_src:
     cluster_profile: aws
+- as: aws-fips-unit
+  commands: GODEBUG=tls13=1 make test-unit
+  openshift_installer_src:
+    cluster_profile: aws
```

Comment:
@petr-muller what go compiler is used in this? er,where/how is this image built? `${LOCAL_IMAGE_SRC}` that's used in the src template? thanks

Reply:
AFAICT, once, `registry.svc.ci.openshift.org/ocp/builder:golang-1.12` is updated with 
`go-toolset-1.12`, the LOCAL_IMAGE_SRC will include the desired go compiler.